lib/service/Service.ts

```diff
@@ -144,26 +144,35 @@ class Service extends EventEmitter {
   }
 
   /**
-   * Get a list of standing peer orders from the order book for a specified trading pair, or for all
-   * trading pairs if no pair is specified.
+   * Get a list of standing peer orders from the order book for a specified trading pair
```

Comment:
Add a period at the end of the comment.

Reply:
Also, it's no longer returning only peer orders so we should remove "peer" here.